cmd/api/main.go

```diff
@@ -34,6 +35,7 @@ const (
 	otelServiceName                   = "kubearchive.api"
 	cacheExpirationAuthorizedEnvVar   = "CACHE_EXPIRATION_AUTHORIZED"
 	cacheExpirationUnauthorizedEnvVar = "CACHE_EXPIRATION_UNAUTHORIZED"
+	enablePprofEnvVar                 = "KUBEARCHIVE_ENABLE_PPROF"
```

Comment:
If you put the pprof setup function into a `pkg` package this `const` could be moved there and then the sink, operator and api could use the shared const